### PR TITLE
package pykicad

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pykicad/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ded058f3af116845d8a0495f3a8ff4b6a7e707e534c5b71ee61b37fb23553923
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
```
Comment (hmaarrfk) [inline recipes/pykicad/meta.yaml]: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python

File: recipes/pykicad/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
+  imports:
+    - pykicad
+
+about:
+  home: https://github.com/dvc94ch/pykicad
+  license: ISC
+  # ISC throws an error if used as the family. I think BSD is appropriate
+  license_family: BSD
```
Comment (scopatz) [inline recipes/pykicad/meta.yaml]: the license family should be OTHER

Comment (hmaarrfk) [reply recipes/pykicad/meta.yaml]: Really?

>  It is functionally equivalent to the simplified BSD and MIT licenses, but without language deemed unnecessary following the Berne Convention.[nb 1][nb 2]

https://en.wikipedia.org/wiki/ISC_license

Comment (scopatz) [reply recipes/pykicad/meta.yaml]: MIT and BSD are not functionally equivalent, so I don't trust this.  It isn't a BSD license

**Comment by scopatz** [comment]
Thanks @hmaarrfk!

**Pull request merged**